modules/ROOT/pages/microsoft/microsoft-365-ops-connector-reference.adoc

```diff
@@ -3,327 +3,317 @@ ifndef::env-site,env-github[]
-| Connection Timeout a| Number |  Specifies the amount of time, in milliseconds, that the consumer attempts to establish a connection before it times out. Default value is 0, which means infinite. |  0 |
-| Host a| String |  Hostname of the proxy. If this property is not set, no proxy is used, otherwise a proxy is used, but a proxy host must be specified. |  |
-| Port a| Number |  Port of the proxy. If host is set then this property must be set and cannot be a negative number. |  |
-| Username a| String |  Username used to authenticate against the proxy. If this property is not set, no authentication is going to be used against the proxy, otherwise this value must be specified. |  |
-| Password a| String |  Password used to authenticate against the proxy. |  |
-|===
-
-== Associated Operations
-
-* executeOperation
-* importDataRecurringJob
-* retrieveMultiple
-* retrieveMultipleByQuery
-* unauthorize
+| Username a| String |  Username used to initialize the session |  | x
+| Password a| String |  Password used to authenticate the user |  | x
+| Resource a| String |  The App ID URI of the web API (secured resource). |  | x
+| Client Id a| String |  The Application ID assigned to your app when you registered it with Azure AD. You can find this in the Azure Portal. Click *Active Directory*, click the directory, choose the application, and click *Configure*. |  | x
+| Client Secret a| String |  The Application Secret that you created in the app registration portal for your app. It should not be used in a native app, because client_secrets cannot be reliably stored on devices. It is required for web apps and web APIs, which have the ability to store the client_secret securely on the server side. |  | x
+| Token Request Endpoint a| String |  |  | x
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+| Read Timeout a| Number |  Specifies the amount of time in milliseconds that the consumer waits for a response before it times out. Default value is 0, which means infinite. |  0 | 
+| Connection Timeout a| Number |  Specifies the amount of time in milliseconds that the consumer attempts to establish a connection before it times out. Default value is 0, which means infinite. |  0 | 
+| Host a| String |  Hostname of the proxy. If this property is not set, then no proxy is used, otherwise a proxy is used, but a proxy host must be specified. |  | 
+| Port a| Number |  Port of the proxy. If host is set then this property must be set and cannot be a negative number. |  | 
+| Username a| String |  Username used to authenticate against the proxy. If this property is not set, then no authentication is going to use against the proxy, otherwise this value must be specified. |  | 
```

Comment:
```suggestion
| Username a| String |  Username used to authenticate against the proxy. If this property is not set, then no authentication is used against the proxy; otherwise this value must be specified. |  | 
```